### PR TITLE
test: results formatter for iac

### PR DESCRIPTION
**Pull request by rontalx**
#### What does this PR do?
Adds a unit test to the results formatter + a minor naming refactor.

### CODE REVIEW

File: src/cli/commands/test/iac-local-execution/results-formatter.ts [hunk 1]
```diff
@@ -15,18 +15,13 @@ import { IacProjectType } from '../../../../lib/iac/constants';
 const SEVERITIES = [SEVERITY.LOW, SEVERITY.MEDIUM, SEVERITY.HIGH];
 
 export function formatScanResults(
-  iacLocalExecutionResults: Array<IacFileScanResult>,
+  scanResults: Array<IacFileScanResult>,
```
Comment (rontalx) [inline src/cli/commands/test/iac-local-execution/results-formatter.ts:18]: changes in this file are just renames

File: test/jest/unit/iac-unit-tests/results-formatter.fixtures.ts [hunk 2]
```diff
@@ -0,0 +1,81 @@
+import {
+  EngineType,
+  IacFileScanResult,
+  PolicyMetadata,
+} from '../../../../src/cli/commands/test/iac-local-execution/types';
+import { IacProjectType } from '../../../../src/lib/iac/constants';
+import { SEVERITY } from '../../../../src/lib/snyk-test/common';
+
+const policyStub: PolicyMetadata = {
+  description: '',
```
Comment (p0tr3c) [inline test/jest/unit/iac-unit-tests/results-formatter.fixtures.ts:10]: remove it, legacy not longer used

File: test/jest/unit/iac-unit-tests/results-formatter.spec.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  scanResults,
+} from './results-formatter.fixtures';
+
+describe('formatScanResults', () => {
+  it('returns the formatted results as expected for output', () => {
+    const formattedResults = formatScanResults(scanResults, {
+      severityThreshold: SEVERITY.HIGH,
+    });
+    expect(formattedResults[0]).toEqual(expectedFormattedResults);
+    expect(formattedResults.length).toEqual(1);
```
Comment (p0tr3c) [inline test/jest/unit/iac-unit-tests/results-formatter.spec.ts:14]: nit: check length first before accessing index.

**Review by p0tr3c — APPROVED** [review]

**Review by ipapast — APPROVED** [review]

File: test/jest/unit/iac-unit-tests/results-formatter.fixtures.ts (hunk 2 above)
Comment (rontalx) [reply test/jest/unit/iac-unit-tests/results-formatter.fixtures.ts]: I know it's considered legacy, but are you sure it's not included anymore in eval output? even if empty?
as the wasm file I'm using for testing still does output this field, for some policies also with information (in the old weird form).

**Pull request merged**